### PR TITLE
Add a reset environment shell script, command and update README.

### PR DESCRIPTION
**Pull request by robbiethegeek-usds**
Add a reset environment shell script, and npm run command and update README.md

### CODE REVIEW

File: script/reset-environment.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+echo "Removing the node modules folder..."
+rm -rf ./node_module
```
Comment (cvalarida) [inline script/reset-environment.sh]: `./node_modules`?

Comment (robbiethegeek-usds) [inline script/reset-environment.sh]: oh the irony of committing my test failures :)

**Review by cvalarida — APPROVED** [review]

**Comment by robbiethegeek-usds** [comment]
@cvalarida can you give it a final LGTM 👍 if you agree?

**Review by cvalarida — APPROVED** [review]
👍 Looks great!

**Pull request merged**